libs/backend/src/cast_vote_records/export.ts

```diff
@@ -668,5 +679,81 @@ export async function exportCastVoteRecordsToUsbDrive(
     return exportSignatureFileResult;
   }
 
+  clearDoesUsbDriveRequireCastVoteRecordSyncCachedResult();
+
```

Comment:
At the end of the export, I assume that we wouldn't require any sync. Why not update the cached value rather than clearing it, as if we don't know?

Reply:
I considered that but ultimately decided that it was safer to just let the `doesUsbDriveRequireCastVoteRecordSync` function recompute, the performance impact being negligible. If we were to update at the end of the export function, we'd have to take extra care to only update if the export was a full export (and not a single-CVR export). Made me realize that updating at the end of export is trickier than it seems and could lead to bugs down the road as the export function is modified.